web/src/shared/components/document-pages/components/document-pdf/index.tsx

```diff
@@ -143,33 +139,41 @@ const DocumentPDF: React.FC<DocumentPDFProps> = ({
 
     // in case of scrolling to some annotation
     useEffect(() => {
-        if (!selectedAnnotation || !pdfPagesListRef.current || !listViewContainerRef.current) {
-            return;
-        }
+        // give the browser chance to make additional painting (to wait for changes with
+        // annotations like creation of new annotation)
+        requestAnimationFrame(() => {
```

Comment:
Ideally, should be refactored in the future to avoid "requestAnimationFrame" at all